conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/TezosNodeInterface.scala

```diff
@@ -1,4 +1,4 @@
-package tech.cryptonomic.conseil.tezos
+package tech.cryptonomic.conseil.common.tezos
```

Comment:
This is almost possible. Currently `NodeStreamingBenchmarks` uses this. Since this class belongs to `Schema` module, it does not have (and should not have) dependency on Lorre. I think we cannot move it yet.

Reply:
The benchmarks are there for historical purpose only, and I suggest you remove them completely if they block us on cleanly separating the modules.
Especially they would need to be refactored anyway to work correctly, I presume, so having them blocking this process makes no sense.

Reply:
For some reason, I was thinking that these benchmarks are part of the `GenSchema` process. I'll remove them and move above class as well.